### PR TITLE
Adding Create ServicesList Airtable Function

### PR DESCRIPTION
**Pull request by btmoore4**

### CODE REVIEW

File: api/airtable.ts [hunk 1]
```diff
@@ -43,4 +43,17 @@ export class AirtableClient {
...
+    id: string,
+    name: string,
+    description: string,
+    url: string, 
+    addresses?: string[],
+    phoneNumbers?: string[], 
+    email?: string, 
+  ): string {
+      return tableName
+  }
```
Comment (dangalan) [inline api/airtable.ts:58]: I think we could make a little more generic? Like just taking in a `tableName` and a list of objects. [Here's Airtable's auto-generating docs for creating a service list for reference.](https://airtable.com/appEQiJGfBTod8EKq/api/docs#javascript/table:services%20lists:create)

Comment (btmoore4) [reply api/airtable.ts:66]: i tried to make it more generic, lmk

File: models/services.ts [hunk 2]
```diff
@@ -4,9 +4,9 @@ export interface Service {
   id: string
   name: string
   description: string
-  address?: string[] // list of IDs linking to locations table
   url: string
   email?: string
+  addresses?: string[] // list of IDs linking to locations table
```
Comment (dangalan) [inline models/services.ts:9]: Any reason for changing from `address` to `addresses`? It looks like the field is still `address` in the dev Airtable

Comment (btmoore4) [reply models/services.ts:9]: Not really lol, I thought about editing some of the other field names too. Fo example some are capitalized and some are not. What do you think? Should I just change it back here, or is it easy enough to just update the dev & prod airtable

Comment (btmoore4) [reply models/services.ts:9]: Eh I guess I should not mess with the actual services table. ServiceLists i don't mind messing with. Will just change it back

Comment (btmoore4) [reply models/services.ts]: Reverted

File: api/airtable.ts (hunk 1 above)
Comment (dangalan) [reply api/airtable.ts:66]: Looks good - we could probably switch the return to `records.map(record => record.fields) as T[]` like in my PR though so we don't need to pass in `keys` anymore

File: package.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "framer-motion": "^4.1.17",
     "geolib": "^3.3.3",
     "next": "^11.1.3",
+    "node-fetch": "^3.2.0",
```
Comment (dangalan) [inline package.json:23]: I think we can remove the this dependency from the `package.json` as long as it's bumped in the `yarn.lock`

**Pull request merged**